app/src/main/java/grinnell/appdev/edu/lyles/fragments/BeersAdapter.java

```diff
@@ -0,0 +1,253 @@
+            super(itemView);
+
+            tvTitle = (TextView) itemView.findViewById(R.id.tvTitle);
+            tvSubtitle = (TextView) itemView.findViewById(R.id.tvSubtitle);
+            tvPrice = (TextView) itemView.findViewById(R.id.tvPrice);
+            tvDetails = (TextView) itemView.findViewById(R.id.tvDetails);
+            ivImage = (ImageView) itemView.findViewById(R.id.ivImage);
+            btFavourite = (ImageButton) itemView.findViewById(R.id.btFavourite);
+
+            itemView.setOnClickListener(new View.OnClickListener(){
+
+                @Override
+                public void onClick(View v) {
+                    if (tvDetails.getVisibility() == View.GONE) {
+                        // it's collapsed - expand it
+                        tvDetails.setVisibility(View.VISIBLE);
+                       // mDescriptionImg.setImageResource(R.drawable.ic_expand_less_black_24dp);
+                    } else {
+                        // it's expanded - collapse it
+                        tvDetails.setVisibility(View.GONE);
+                        //mDescriptionImg.setImageResource(R.drawable.ic_expand_more_black_24dp);
+                    }
+
+                    ObjectAnimator animation = ObjectAnimator.ofInt(tvDetails, "maxLines", tvDetails.getMaxLines());
+                    animation.setDuration(200).start();
+
+                }
+            });
+        }
+    }
+    // ... view holder defined above...
+
+    // Store a member variable for the contacts
+    private List<Beers> mBeers;
+    // Store the context for easy access
+    private Context mContext;
+
+    // Pass in the contact array into the constructor
+    public BeersAdapter(BeerFragment context, List<Beers> Beers) {
+        mBeers = Beers;
+        mContext = context.getActivity().getApplicationContext();
+    }
+
+    // Easy access to the context object in the recyclerview
+    private Context getContext() {
+        return mContext;
+    }
+
+    // ... constructor and member variables
+
+    // Usually involves inflating a layout from XML and returning the holder
+    @Override
+    public BeersAdapter.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        Context context = parent.getContext();
+        LayoutInflater inflater = LayoutInflater.from(context);
+
+        // Inflate the custom layout
+        View beerView = inflater.inflate(R.layout.beer_row, parent, false);
+
+        // Return a new holder instance
+        ViewHolder viewHolder = new ViewHolder(beerView);
+        return viewHolder;
+    }
+
+    // Involves populating data into the item through holder
+    @Override
+    public void onBindViewHolder(BeersAdapter.ViewHolder viewHolder, int position) {
+        // Get the data model based on position
+        Beers beer = mBeers.get(position);
+
+        // Set item views based on your views and data model
+        TextView tvTitle = viewHolder.tvTitle;
+        tvTitle.setText(beer.getTitle());
```

Comment:
You don't need to make a variable for each of these. For example, this line and the one above it could be condensed to `viewHolder.tvTitle.setText(beer.getTitle());` without losing much readability.